playbooks/secrets.yml

```diff
@@ -14,6 +14,10 @@
           If you are upgrading from a previous playbook version, please read to docs/playbook-upgrade.md"
         quiet: true
 
+- name: Autogenerate secrets if this is a testing environment
+  ansible.builtin.import_playbook: secrets-init.yml
+  when: test_environment is defined and test_environment|bool
```

Comment:
Please a choose a variable name that really sas what the param does. `test_environment`sounds like an option that's not wise to use.
I had in mind stuff like `autogen_secrets` or similar.

Reply:
I think about it twice and it's better if it's self-explanatory, for better understanding of everyone

Reply:
Do you mean `test_environment` is self-explainatory?
If so, I have to disagree on that. allowing using autogenerated secrets is not linked only to test scenarios

Reply:
sorry, the subject was your proposal, I am switching to `autogen_unsecure_secrets` to enforce that is not a good practice to enable it